src/views/HomeView.vue

```diff
@@ -1,7 +1,23 @@
 <template>
```

Comment:
Maybe change the view name to QuizView? Including changes to the routes and such